notebooks/1.2-ijd-fetch-circle-elevations.ipynb

```diff
@@ -45,7 +45,7 @@
   },
```

Comment:
<p>These outputs don't match. Maybe not a problem, though....because the dataset is a bit different without the duplicates.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/DataKind-DC/audubon-cbc/pull/59/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Reply:
<p>I have not yet re-run to create a new offline file. i'll start it now so you can review with the most up to date data. </p>